python/interpret_community/common/explanation_utils.py

```diff
@@ -17,8 +17,10 @@
 with warnings.catch_warnings():
     warnings.filterwarnings('ignore', 'Starting from version 2.2.1', UserWarning)
     import shap
-    from shap.common import DenseData
-
+    try:
+        from shap.common import DenseData
+    except:
+        from shap.utils._legacy import DenseData
```

Comment:
Why is DenseData being removed from the public shap interface? What does the shap package recommend that dependent packages use instead?

Reply:
shap is moving to a new API, dense data was always just an alternative to numpy/pandas inputs, it seems they decided to fully remove it

Reply:
Can we take a dependency on numpy/pandas rather than relying on the private _legacy module? We don't want to take a dependency on deprecated functionality